wolfssl/internal.h

```diff
@@ -6960,7 +7013,7 @@ WOLFSSL_LOCAL int tlsShowSecrets(WOLFSSL* ssl, void* secret,
 /* Optional Pre-Master-Secret logging for Wireshark */
 #if !defined(NO_FILESYSTEM) && defined(WOLFSSL_SSLKEYLOGFILE)
 #ifndef WOLFSSL_SSLKEYLOGFILE_OUTPUT
-    #define WOLFSSL_SSLKEYLOGFILE_OUTPUT "sslkeylog.log"
+    #define WOLFSSL_SSLKEYLOGFILE_OUTPUT "/tmp/secrets"
```

Comment:
why?

Reply:
This is what I use for wireshark and committed by accident. Restoring